src/engine/strat_engine/blockdevmgr.rs

```diff
@@ -364,8 +365,11 @@ fn initialize(pool_uuid: PoolUuid,
             let allocator = RangeAllocator::new(bda.dev_size(), &[(Sectors(0), bda.size())])
                 .expect("bda.size() < bda.dev_size() and single range");
 
-            // TODO: support getting hw info and passing in here. See #615
-            bds.push(StratBlockDev::new(dev, devnode.to_owned(), bda, allocator, None, None));
+            let hw_id = match hw_lookup(devnode) {
```

Comment:
Yes. This is exactly the place where we would like to log because we ought not to fail, as the value is optional, but we would like to know about the problem. Feel free to put a TODO here, probably right next to the error.